.gitignore

```diff
@@ -1,3 +1,5 @@
 corectl
 
 dist/
+.idea/
```

Comment:
Christopher asked not to add environment-specific gitignore entries. You can add those to your personal gitignore.
https://dev.to/fronkan/a-personal-gitignore-even-for-a-single-repository-4o7h

Reply:
done

Reply:
Have you pushed the changes?